services/time/src/lib.rs

```diff
@@ -115,22 +116,22 @@ impl Transaction for TxTime {
         self.verify_signature(self.pub_key())
     }
 
-    fn execute(&self, view: &mut Fork) {
+    fn execute(&self, view: &mut Fork) -> ExecutionResult {
         let validator_keys = Schema::new(&view).actual_configuration().validator_keys;
 
         // The transaction must be signed by the validator.
         let signed = validator_keys.iter().any(|&validator| {
             validator.service_key == *self.pub_key()
         });
         if !signed {
-            return;
+            return Ok(());
         }
 
         let mut schema = TimeSchema::new(view);
         match schema.validators_times().get(self.pub_key()) {
             // The validator time in the storage should be less than in the transaction.
             Some(storage_time) if storage_time >= self.time() => {
-                return;
+                return Ok(());
```

Comment:
Now, this case is interesting. On the one hand, it is sort of unexpected that the storage time of a validator decreases. OTOH, it's not obvious that such a transaction should not put changes into the storage.